webapp/src/shared/components/Text/Text.tsx

```diff
@@ -0,0 +1,37 @@
+import { Color } from '../../types';
+
+export enum TextSize {
+  EXTRA_SMALL = '0.75rem',
+  SMALL = '0.875rem',
+  SMALL_MEDIUM = '1rem',
+  MEDIUM = '1.25rem',
+  MEDIUM_LARGE = '1.5rem',
+  LARGE = '2.5rem',
+  EXTRA_LARGE = '3rem',
+}
+
+export enum TextWeight {
+  REGULAR = '400',
+  MEDIUM = '500',
+  BOLD = '700',
+}
+
+type TextProps = {
+  children: string;
+  size?: TextSize;
+  color?: Color;
+  weight?: TextWeight;
+};
+
+export default function Text({
+  size = TextSize.MEDIUM,
+  color = Color.DARK,
+  weight = TextWeight.REGULAR,
+  children,
+}: TextProps) {
+  return (
+    <div style={{ fontSize: size, color: `var(${color})`, fontWeight: weight }}>
+      {children}
+    </div>
```

Comment:
> What about the font family. I think it currently uses the font it inherits from the parent component. Should we add a prop to change the font, or maybe something else?

I believe the font family should be a root property of our app because it is really tight with all the design. It wouldn't make sense to have several font types around the app, so I'd say inheriting from the parent component is perfectly ok. The font-family should be defined as root in css

Reply:
What do we think about using another name for the `type`, for example, `variant`? Because `type` is a TypeScript keyword, the line `const TextTag = type as React.ElementType` is a bit confusing for me.

Reply:
> What do we think about using another name for the type, for example, variant? Because type is a TypeScript keyword, the line const TextTag = type as React.ElementType is a bit confusing for me.

agree with this

Reply:
Done :+1: